front/components/cart/cart.html

```diff
@@ -18,10 +18,11 @@
       <td><input type="text" ng-module="{{product.qty}}"> change qty</td>
     </tr>
   </table>
-  current total:{{total}}<br>
 
-  <input type="submit" value"delete" ng-click="delete()">
-  <input type="submit" value"update" ng-click="update()">>
+  <b>current total:{{total}}</b><br>
+
+  <input type="submit" value="delete" ng-click="delete()">
```

Comment:
Not much point in using an input here.  You should probably use a <button></button> instead